### PR TITLE
Fixes for newer PWSH + K4 tables

### PR DESCRIPTION
**Pull request by avbenavides**
Added -SkipHeaderValidation apeace newer PWSH
Added K4 tables at the end of the EtradeK4.ps1

### CODE REVIEW

**Review by matt77hias — CHANGES_REQUESTED** [review]
Looks all good to me.
Could you only revise the space vs tab mixing?

File: Scripts/EtradeK4.ps1 [hunk 1]
```diff
@@ -39,6 +39,68 @@ $gainsLosses = $(get-content -raw -path gainsLosses.json | ConvertFrom-Json).dat
...
+		$txDate = [datetime]::ParseExact($ct.closingTransDateSold, "MM/dd/yyyy", $null)
+		$txRate = Get-SEKUSDPMIAtDate($txDate)
+		
+		$tx = $ct.closingTransTotalProceeds
+		$txSEK = [int][math]::Round($tx * $txRate)
+		
+		$closingTransTotalProceeds = [math]::Max(0.0, $tx)
+		$closingTransTotalProceedsSEK = [math]::Max(0, $txSEK)
+
+        $txDate = [datetime]::ParseExact($ct.openingTransDateAcquired, "MM/dd/yyyy", $null)
```
[the remark this answers is not in the document]
Comment (avbenavides) [reply Scripts/EtradeK4.ps1:59]: I'm a spaceman!
I'll fix it

**Comment by avbenavides** [comment]
Update spaces to tabs @matt77hias

**Review by matt77hias — APPROVED** [review]

**Comment by matt77hias** [comment]
Thank you for your contributions! @avbenavides

**Pull request merged**